sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/reflection/KalixMethod.scala

```diff
@@ -42,6 +42,7 @@ object ServiceMethod {
   // this is more for early validation. We don't support stream-in over http,
   // we block it before deploying anything
   def isStreamIn(method: Method): Boolean = {
+    // TODO: remove this once is validations completely moved to Validations object
```

Comment:
```suggestion
```
In case all validations have been moved to Validations

Reply:
That will probably be done in follow up PRs